shared/styles/style-guide.desktop.js

```diff
@@ -67,19 +69,30 @@ const fontCommon = {
 const font = {
   fontRegular: {
     ...fontCommon,
-    fontFamily: 'Noto Sans'
+    fontFamily: 'Lato',
+    fontWeight: 400
   },
-  fontBold: {
+  fontSemibold: {
     ...fontCommon,
-    fontFamily: 'Noto Sans Bold'
+    fontFamily: 'Lato',
```

Comment:
Won't this switch the fonts of anyone already using this (non-dz2 stuff) doesn't this need to be under a feature flag?